sdk/formrecognizer/ai-form-recognizer/samples/javascript/recognizeReceipt.js

```diff
@@ -37,40 +37,35 @@ async function main() {
     throw new Error("Expecting at lease one receipt in analysis result");
   }
 
-  const usReceipt = receipts[0];
+  const receipt = receipts[0];
   console.log("First receipt:");
-  console.log(`Receipt type: ${usReceipt.receiptType.type} with confidence ${usReceipt.receiptType.confidence}`);
-  console.log(
-    `Merchant Name: ${usReceipt.merchantName.value} (confidence: ${usReceipt.merchantName.confidence})`
-  );
-  console.log(
-    `Transaction Date: ${usReceipt.transactionDate.value} (confidence: ${usReceipt.transactionDate.confidence})`
-  );
-  console.log("Receipt items:");
-  console.log(`  name\tprice\tquantity\ttotalPrice`);
-  for (const item of usReceipt.items) {
-    const name = `${optionalToString(item.name.value)} (confidence: ${optionalToString(
-      item.name.confidence
-    )})`;
-    const price = `${optionalToString(item.price.value)} (confidence: ${optionalToString(
-      item.price.confidence
-    )})`;
-    const quantity = `${optionalToString(item.quantity.value)} (confidence: ${optionalToString(
-      item.quantity.confidence
-    )})`;
-    const totalPrice = `${optionalToString(item.totalPrice.value)} (confidence: ${optionalToString(
-      item.totalPrice.confidence
-    )})`;
-    console.log(`  ${name}\t${price}\t${quantity}\t${totalPrice}`);
+  const receiptTypeField = receipt.recognizedForm.fields["MerchantName"];
```

Comment:
Consider adding a link to https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult to show users the fields available

Reply:
Done. Yes this is very helpful!